setup.json

```diff
@@ -76,7 +76,7 @@
         ],
```

Comment:
With ``pylint==2.5.2`` I get the following error: 

```
$ pylint --version
Traceback (most recent call last):
  File "/home/a-dogres/.virtualenvs/aiida-qe/bin/pylint", line 8, in <module>
    sys.exit(run_pylint())
  File "/home/a-dogres/.virtualenvs/aiida-qe/lib/python3.7/site-packages/pylint/__init__.py", line 22, in run_pylint
    PylintRun(sys.argv[1:])
  File "/home/a-dogres/.virtualenvs/aiida-qe/lib/python3.7/site-packages/pylint/lint/run.py", line 306, in __init__
    linter.load_config_file()
  File "/home/a-dogres/.virtualenvs/aiida-qe/lib/python3.7/site-packages/pylint/config.py", line 785, in load_config_file
    for option, value in parser.items(section):
  File "/usr/lib/python3.7/configparser.py", line 857, in items
    return [(option, value_getter(option)) for option in d.keys()]
  File "/usr/lib/python3.7/configparser.py", line 857, in <listcomp>
    return [(option, value_getter(option)) for option in d.keys()]
  File "/usr/lib/python3.7/configparser.py", line 854, in <lambda>
    section, option, d[option], d)
  File "/usr/lib/python3.7/configparser.py", line 394, in before_get
    self._interpolate_some(parser, option, L, value, section, defaults, 1)
  File "/usr/lib/python3.7/configparser.py", line 411, in _interpolate_some
    p = rest.find("%")
AttributeError: 'list' object has no attribute 'find'
```

I think it is this issue, which is fixed in ``2.5.3``: https://github.com/PyCQA/pylint/issues/3538

Since the pre-commit appears to pin ``2.5.2``, I don't know why it doesn't show up in CI.

Reply:
I think it would make sense to _pin_ the pylint version also here (probably to 2.6.0), because the messages change between versions.